### PR TITLE
add managedBy support

### PR DESCRIPTION
**Pull request by sbaerlocher**
##### SUMMARY
Function implementation for the Issues #173 

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
win_domain_computer

### CODE REVIEW

**Comment by briantist** [comment]
@sbaerlocher if you phrase it as fixes #173 or resolves #173 GitHub will automatically link these and ensure that merging the PR closes the issue.

**Review by jborean93 — CHANGES_REQUESTED** [review]
As per @briantist's comments you need a changelog fragment. You also need to add the new option to the module docs so people can read more about it.

One thing to note in the PR is there are a few style changes. It would be great if you could revert those changes and keep the existing code the way it was. It makes it a lot harder to sort out actual code changes compared to just style changes when reviewing and breaks the git attribution info for that line for no real reason.

**Comment by goshansp** [comment]
@sbaerlocher thanks alot for your work! i was not able to succeed in a first quick round of testing. the object is created, but the managedBy-field is not populated in the AD and it gives no error. also we seem to lack documentation - i will resume testing next week hoping to contribute. thanks!

**Comment by sbaerlocher** [comment]
@goshansp I have revised and tested it again and adapted the documentation, can you test it again?

**Comment by goshansp** [comment]
creating (New-) and updating (Set-) have both been successfully tested against our AD - good work @sbaerlocher, thanks! i added updated the changelog and it can be shipped! can you estimate when this feature will be released?

File: plugins/modules/win_domain_computer.py [hunk 1]
```diff
@@ -86,6 +86,12 @@
     type: str
     choices: [ absent, present ]
     default: present
+  managed_by:
+    description:
+    - The value to be assigned to the LDAP C(managedBy) attribute.
+    - This value can be in the forms C(Distinguished Name), C(objectGUID),
+      C(objectSid) or C(sAMAccountName), see examples for more details.
+    type: str
```
Comment (briantist) [inline plugins/modules/win_domain_computer.py:94]:
```suggestion
  managed_by:
    description:
    - The value to be assigned to the LDAP C(managedBy) attribute.
    - This value can be in the forms C(Distinguished Name), C(objectGUID),
      C(objectSid) or C(sAMAccountName), see examples for more details.
    type: str
    version_added: '1.3.0'
```
I think you'll need the field for when this option was added. I'm also not certain what the next version of the collection is; current is 1.2.0, so I'm taking a guess at 1.3.0.

**Comment by sbaerlocher** [comment]
I have now adjusted the following:
- Reset the style.
- The changelog supplemented.
- version_add information added (have now added times 1.3.0, @jborean93 can still tell me the release?)

The changes were tested by me and @goshansp.

@jborean93 can you please check it and tell me in what period it will be released in the collection?

File: plugins/modules/win_domain_computer.py (hunk 1 above)
Comment (sbaerlocher) [reply plugins/modules/win_domain_computer.py:94]: version_add information added

**Review by jborean93 — APPROVED** [review]

**Comment by jborean93** [comment]
Changes look great, thanks for working on it

> @jborean93 can you please check it and tell me in what period it will be released in the collection?

I've doing a pass on all the PRs this week with the aim of creating 1.3.0 something at the end of the week or sometime in next week.

**Comment by goshansp** [comment]
thank you very much @jborean93 @keilr @sbaerlocher for making this happen! the customer will be very pleased.

@jborean93 can you estimate by when this will be available in ansible=2.10.6 on pypi? or will it have to wait for 2.11? i'd be happy to test the ansible-rc once we have it on pypi.

**Comment by jborean93** [comment]
@goshansp I'm not sure what the release schedule is for ansible 2.10 but this will be present in the 2.10.x release after community.windows is released with this fix.

**Pull request merged**